Source/PLCrashReport.h

```diff
@@ -190,6 +194,17 @@ typedef struct _PLCrashReportDecoder _PLCrashReportDecoder;
  */
 @property(nonatomic, readonly, strong) PLCrashReportExceptionInfo *exceptionInfo;
 
+/**
+ * YES if user information is available.
+ */
+@property(nonatomic, readonly) BOOL hasUserInfo;
```

Comment:
This flag looks redundant, I guess `nil` check of the main field does the same

Reply:
I wrote it because there were other properties similar to this, but I will delete